Source/Foundation/bsfUtility/Private/UnitTests/BsFileSystem_test.cpp

```diff
@@ -0,0 +1,274 @@
+//************************************ bs::framework - Copyright 2018 Marko Pintera **************************************//
+//*********** Licensed under the MIT license. See LICENSE.md for full terms. This notice is not to be removed. ***********//
+
+#include "Testing/BsTestSuite.h"
```

Comment:
Would be nice it we didn't lose git history for these files, and could just see the changes done to accommodate google test.

Reply:
eh the only changes were changing BS_ASSERT to ASSERT_TRUE or ASSERT_EQ.